core/src/executor/evaluate/error.rs

```diff
@@ -61,3 +64,60 @@ pub enum EvaluateError {
     #[error("format function does not support following data_type: {0}")]
     UnsupportedExprForFormatFunction(String),
 }
+
+#[derive(Error, Serialize, Debug, PartialEq)]
+pub enum ChronoFormatError {
```

Comment:
```rust

impl ChronoFormatError {
    pub fn err_into(error: chrono::format::ParseError) -> crate::result::Error {
        let error: ChronoFormatError = error.into();
        let error: EvaluateError = error.into();
        error.into()
    }
}
```

Let's implement this to improve our two-step `into` operation.

Reply:
Well... the question may be where is the definition for this.
Even if it exists as a single function, there is no problem. If there is a suitable place, please change it there.